package.json

```diff
@@ -1,16 +1,17 @@
 {
-  "name": "@smg-automotive/example",
+  "name": "@smg-automotive/next-page-bundlesize",
   "version": "0.0.0-development",
```

Comment:
Update?

Reply:
no, semantic release controls the version